README.md

```diff
@@ -85,9 +76,11 @@ func main() {
 import "github.com/hellofresh/stats-go/bucket"
 
 timing := statsClient.BuildTimer().Start()
-operations := bucket.MetricOperation{"orders", "order", "create"}
+operation := bucket.MetricOperation{"orders", "order", "create"}
 err := orderService.Create(...)
-statsClient.TrackOperation("ordering", operations, timing, err == nil)
+statsClient.TrackOperation("ordering", operation, timing, err == nil)
```

Comment:
Struct is fine when you have optional params, otherwise it makes code more verbose, that is not good for frequent calls - this is just a method call that does something with all passed parameters. Moreover, you need to allocate/clean memory more frequently in this case, that does not make sense for frequently called simple piece of code.

Moving to config was good idea for `bucket.NewHasIDAtSecondLevelCallback()` - you can init single field and do not care about the rest. And this is called once, so verbosity does not add much to overall code.